lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -26,11 +26,34 @@ const log = require('../../../lighthouse-core/lib/log');
 const ReportGenerator = require('../../../lighthouse-core/report/report-generator');
 
 const STORAGE_KEY = 'lighthouse_audits';
-const _flatten = arr => [].concat(...arr);
 
+let installedExtensions = [];
 let lighthouseIsRunning = false;
 let latestStatusLog = [];
 
+const _flatten = arr => [].concat(...arr);
+
+/**
+ * Enables or disables all other installed chrome extensions. The initial list
+ * of the user's extension is created when the background page is started.
+ * @param {!boolean} enable If true, enables all other installed extensions.
+ *     False disables them.
+ * @param {!Promise}
+ */
+function enableOtherChromeExtensions(enable) {
+  return new Promise(resolve => {
```

Comment:
will you add a `log.log` or `log.verbose` about enabling or disabling extensions? Worth including `installedExtensions.length` in the log message?